demos/common/tcp/aws_tcp_echo_client_single_task.c

```diff
@@ -338,7 +338,7 @@ static void prvEchoClientTask( void * pvParameters )
                      * length. */
                     ulTxRxFailures[ xInstance ]++;
                     configPRINTF( ( "ERROR: xTransmitted %d xReceivedBytes %d \r\n",
-						(int)xTransmitted, (int)xReceivedBytes ) );
+                                    (int)xTransmitted, (int)xReceivedBytes ) );
```

Comment:
This update is still failing the style checks because we we need the spaces inside the parentheses on the casts.